packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -34,32 +34,27 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable {
     bytes32 public constant defaultIdentifier = "ASSERT_TRUTH";
 
     IERC20 public defaultCurrency;
-    uint256 public defaultBond;
     uint256 public defaultLiveness;
 
+    CachedUmaParams public cachedUmaParams;
```

Comment:
Not sure about this but: should we add a getter returning the struct to be able to read the nested mappings?

Reply:
I was also not sure and mostly had this issue when writing tests.